src/main/java/com/google/sps/data/User.java

```diff
@@ -3,14 +3,25 @@
 import java.util.Collection;
 import java.util.HashSet;
 
+
 public final class User {
+  public static enum profileType {
```

Comment:
enum types are classes. Please capitalize the enum name following the style guide recommendations.

Reply:
Done